### PR TITLE
PHPLIB-786: Rename "Versioned API" to "Stable API" in docs

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/PHPLIB-786

### CODE REVIEW

File: docs/tutorial.txt [hunk 1]
```diff
@@ -15,4 +15,4 @@ Tutorials
    /tutorial/indexes
    /tutorial/tailable-cursor
    /tutorial/example-data
-   /tutorial/versioned-api
+   /tutorial/stable-api
```
Comment (jmikola) [inline docs/tutorial.txt:18]: @mongodb/docs-platform Is there an easy way to implement a web redirect from `versioned-api` to `stable-api`? Or is this something the docs team would handle themselves in https://github.com/mongodb/docs-php-library?

I'm not terribly concerned about this, as I doubt we have many direct links to this tutorial out there.

File: docs/tutorial/stable-api.txt [hunk 2]
```diff
@@ -43,10 +43,9 @@ Strict API
 ----------
 
 By default, declaring an API version guarantees behavior for commands that are
-part of the versioned API, but does not forbid using commands that are not part
+part of the stable API, but does not forbid using commands that are not part
 of the API version. To only allow commands and options that are part of the
-versioned API, specify the ``strict`` option when creating the
-specify the ``strict`` option when creating the
```
Comment (jmikola) [inline docs/tutorial/stable-api.txt:49]: This appears to have been a typo dating back to the original commit for this page (21e7ecde42e45d98ced06c6818495289c1e2e916).

File: docs/tutorial.txt (hunk 1 above)
Comment (schmalliso) [reply docs/tutorial.txt:18]: It would normally be handled in the docs-php-library repo  in the config/redirects file!

Comment (jmikola) [reply docs/tutorial.txt:18]: @schmalliso: Thanks, I'll make a note of this after merging and opening a DOCSP ticket requesting a deployment.

**Pull request merged**